docs/first-steps/tracking/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Tracking your first events"
+sidebar_position: 3
+sidebar_label: "Tracking events"
+description: "Tracking your first Snowplow events"
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+Once your pipeline is set up, you will want to send some events to it. Here’s an overview of the different options.
+
+:::note Limits
+
+Keep in mind that some of our offerings have limits on the number of events you can send:
+* For BDP Cloud, there is a default cap of 60 events per second and a total of 80m events per month
```

Comment:
I would remove this as I wouldn't want someone to disqualify themselves from BDP Cloud because of it - we are updating the pricing calculator shortly to make it clear you can track more but need to get in touch with us first